.github/workflows/swiftlint.yml

```diff
@@ -0,0 +1,42 @@
+name: SwiftLint
+
+on:
+  push:
+    branches: stable
+    paths:
+      - '.github/workflows/swiftlint.yml'
+      - '.swiftlint.yml'
+      - '**/*.swift'
+  pull_request:
+    paths:
+      - '.github/workflows/swiftlint.yml'
+      - '.swiftlint.yml'
+      - '**/*.swift'
+
+env:
+  xcode-version: 12.0.0
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: macos-latest
+    steps:
+      - 
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Xcode Select
+        uses: devbotsxyz/xcode-select@v1.0.0
+        with:
+          version: ${{ env.xcode-version }}
```

Comment:
I haven't double-checked our Rakefile but do we actually need that, as opposed to our Rakefile prepending something like `DEVELOPER_DIR=$(find 12.0.0 version of Xcode via mdfind)`?

Reply:
May be a leftover from when I had this action directly invoke the SwiftLint script (sh), will have to check.